libs/entry-components/search-filter/date-time/date-time-search-filter.component.html

```diff
@@ -1,6 +1,7 @@
-<mat-form-field [formGroup]="form" subscriptSizing="dynamic">
+<mat-form-field class="date-time-search-field" [formGroup]="form" subscriptSizing="dynamic">
     <mat-label [attr.for]="searchFilter.key">{{searchFilter.label}}</mat-label>
     <input matInput [formControlName]="searchFilter.key" [matDatepicker]="picker" [id]="searchFilter.key">
     <mat-datepicker-toggle matIconSuffix [for]="picker"></mat-datepicker-toggle>
     <mat-datepicker #picker></mat-datepicker>
+    <mat-error entryDisplayControlValidation [control]="form.get(searchFilter.key)"></mat-error>
```

Comment:
@msljivic Just a minor comment, but what do you think about having a control that will be deriving from control value accessor? This would allow us to use [formControlName]="searchFilter.key" syntax. 
E.g.
`<entry-form-field-error [formControlName]="searchFilter.key"/>`

Reply:
It would not be a appropriate use of CVA. Directive `entryDisplayControlValidation` seems fine, it's used only to display validation error so it's applied to <mat-error> element.

If we would implement CVA, implemented methods (writeValue, registerOnChange....) would have no purpose.

Reply:
Usually errors are displayed next to input. So control that wraps input and validation display (e.g. entry-input) would implement CVA

Reply:
My motivation comes from writing more condensed, readable, maintainable code. 

This is way more readabable and thus more difficult to make a mistake:
`<entry-form-field-error [formControlName]="searchFilter.key"/>`
compared to this:
`<mat-error entryDisplayControlValidation [control]="form.get(searchFilter.key)"></mat-error>`

Is there another way to achieve this condensity? I do favor readability over "correctness" of implementation. I'm not even sure that I agree that this is "correct".

Reply:
@msljivic what do you think about this as an alternative:
`<entry-error-display [control]="form.get('field')"></entry-error-display>`

And component:
```
@Component({
  selector: 'entry-error-display',
  template: `
    <mat-error *ngIf="control.invalid && (control.dirty || control.touched)">
      <!-- Add other error messages based on control.errors -->
    </div>
  `
})
export class EntryErrorDisplayComponent {
  @Input() control: AbstractControl;
}
```

Reply:
Component alternative is ok. We can do it that way.